auto-complete-plugin/src/packages/QuestionAutoComplete.vue

```diff
@@ -41,6 +42,13 @@ const Inquirer_Default_Separator = "\u001b[2m───────────
 export default {
   name: "QuestionAutocomplete",
   methods: {
+    onUpdateMenu(open) {
+      if (open) {
+        // WORKAROUND: fixes dialog menu popup position on first click
+        // Issue: https://github.com/vuetifyjs/vuetify/issues/17126 
+        setTimeout(() => window.dispatchEvent(new Event("resize")), 0);
+      }
+    },
```

Comment:
note: I saw this workaround and wasn't too sure if it would of been acceptable. Just because you can sometimes see the menu slide over.

Reply:
You are right it is not so great behavior because at first click you can see this slide, but it is something we can maybe live with until vuetify will fix it.